sdk/sdk-view/src/test/java/mil/emp3/core/StorageManagerTest.java

```diff
@@ -0,0 +1,462 @@
+package mil.emp3.core;
```

Comment:
Many things you sate above already have associated unit tests. If there aren't then we need to add them. The test we need is to make sure that filtering is actually happening when user specifies it. Also what happens in a multi thread environment.